osu.Game.Rulesets.Tau/Objects/Drawables/DrawableSlider.cs

```diff
@@ -75,6 +75,8 @@ public DrawableSlider(Slider obj)
                 slidingSample = new PausableSkinnableSound { Looping = true }
             });
 
+            path.Ticks = repeatContainer;
```

Comment:
I'm not sure if we'd want to go with this name, as it may be confused with actual slider "ticks" (non-repeats).

Reply:
but they are actual slider ticks. So it's a slider tick.... wait... thanks for reminding me